### ISSUE TITLE
Blog posts do not have individual pages

### ISSUE THREAD

**Issue by thegsi** [koalabears/koalabears.github.io#60]
Blog posts do not have their own individual pages.

Main page contains all blog post information and no links to individual posts.

Archive may become difficult to examine.

What does this mean for SEO, users navigation?

**Comment by des-des** [comment]
Agreed we need to do something with this. Would like to find a more dynamic way for the user to navigate blogs (but maybe within our main page) - still not convinced by need for separate pages. Blogs also have there own url but they are not linked atm..
